packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -511,12 +511,38 @@ export const getSignatureWithNote = (name: string, note: string) => {
   return `${getSignature(name)}: ${note}\n`;
 };
 
-export const userCanVote = (user: UsersMinimumInfo|DbUser) => {
-  if (!isLW) return true;
+export const userCanVote = (user: UsersMinimumInfo|DbUser|null): {
+  canVote: true
+  whyYouCantVote?: never
+} | {
+  canVote: false
+  whyYouCantVote: string
```

Comment:
You should use [PermissionResult](https://github.com/ForumMagnum/ForumMagnum/blob/130665f37c872a6657456602aef16128840e0d3c/packages/lesswrong/lib/make_voteable.ts#L6), I claim

Reply:
Seems sensible